### PR TITLE
Corrected conversion factor for lbf (Issue 0004586)

### PR DESCRIPTION
**Pull request by davesrocketshop**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [x ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Base/Quantity.cpp [hunk 1]
```diff
@@ -299,7 +299,7 @@ Quantity Quantity::Ounce            (0.0283495231  ,Unit(0,1));
 Quantity Quantity::Stone            (6.35029318    ,Unit(0,1));
 Quantity Quantity::Hundredweights   (50.80234544   ,Unit(0,1));
 
-Quantity Quantity::PoundForce       (224.81        ,Unit(1,1,-2)); // Newton  are ~= 0.22481 lbF
+Quantity Quantity::PoundForce       (4448.22       ,Unit(1,1,-2)); // lbF are ~= 4.44822 Newton
```
Comment (hyarion) [inline src/Base/Quantity.cpp:302]: While you're at it, maybe you could change `lbF` to `lbf`, [to make it match with the unit you write](https://github.com/FreeCAD/FreeCAD/blob/master/src/App/ExpressionParser.l#L244).
```suggestion
Quantity Quantity::PoundForce       (4448.22       ,Unit(1,1,-2)); // lbf are ~= 4.44822 Newton
```

**Review by hyarion — CHANGES_REQUESTED** [review]
Nice catch!
I've tested it and it was indeed wrong.
The change matches other approximations of the conversion.

While testing I noticed a typo in the unit in the comment, maybe you could change that too?
I've added a suggestion you can apply to fix it.

**Comment by davesrocketshop** [comment]
Done. I actually waffled about making that change earlier

**Review by hyarion — APPROVED** [review]
Perfect! 👍

**Comment by wwmayer** [comment]
Merged as 5edd7d01a0

**Pull request closed without merging**